app/components/markdown/markdown.tsx

```diff
@@ -471,10 +473,14 @@ const Markdown = ({
     };
 
     const renderText = ({context, literal}: MarkdownBaseRenderer) => {
+        const selectable = (managedConfig.copyAndPasteProtection !== 'false') && context.includes('table_cell');
```

Comment:
```suggestion
        const selectable = (managedConfig.copyAndPasteProtection !== 'true') && context.includes('table_cell');
```

When enabled user cannot use copy/paste

Reply:
I copy pasted the logic from somewhere else and did it wrong :sweat_smile:

Just to confirm (and fix in in place), are these two places also doing it wrong?
https://github.com/mattermost/mattermost-mobile/blob/a7db26992dcc704c76200ee24e71f27d0da61468/app/screens/code/index.tsx#L34
Here should be also `!== 'true'`?

https://github.com/mattermost/mattermost-mobile/blob/a7db26992dcc704c76200ee24e71f27d0da61468/app/components/syntax_highlight/renderer.tsx#L77
Here should be the `selectable` argument instead of `true`?

Reply:
Yes they are.. we want to be able to select and copy paste when there is no copy and paste protection